integration-cli/docker_cli_help_test.go

```diff
@@ -15,7 +16,13 @@ func TestMainHelpWidth(t *testing.T) {
 
 	var home string
 	if runtime.GOOS != "windows" {
-		home = os.Getenv("HOME")
+		// Only do this on non-windows systems
+		home = homedir.Get()
+
+		// Don't check when HOME is /
+		if home == "/" {
+			home = ""
```

Comment:
@duglin okay, now I get it, my bad. I was testing your test on top of master, to see if the coverage increased. But the test passes on master because it was a false positive. So if I only checkout the pkg/mflag change from this PR and run the old test from master, I get an error, but only if I set HOME=/.

Maybe we should temporarily set HOME to / in the test?